docs/content/docs/build/audience.md

```diff
@@ -14,18 +14,12 @@ This document will explain how to use the audience APIs and then provide example
 
 When creating a container, you are also provided a container services object which holds the audience.  This audience is backed by that same container.
 
-```typescript
+```js
 const { container, containerServices } =
     await tinyliciousClient.createContainer(serviceConfig, containerSchema);
 const audience = containerServices.audience;
 ```
 
-{{% callout tip %}}
-
-The backing container controls the audience by adding and removing members as part of processing [signals]({{< relref signals.md >}}).  This means audience membership reflects the container's processed signals rather than live information from the service, and delays in signal receipt or processing may also produce outdated audience information.
-
-{{% /callout %}}
-
```

Comment:
I don't think this is super-relevant, and it is confusing because my immediate question is, "So how do I get 'live information from the service?'"

Reply:
There are some non-audience-specific nuances around this system that don't need to be covered in this article, but it would be nice if there's a link to the article that does cover them (which probably doesn't exist yet)

Reply:
Yeah, when we are ready to write the advanced stuff, then we can link to it here.